loaders/httpfs/loader.go

```diff
@@ -0,0 +1,30 @@
+package httpfs
+
+import (
+	"io"
+	"net/http"
+)
+
+type httpFileSystemLoader struct {
+	http.FileSystem
+}
+
+func NewLoader(fs http.FileSystem) httpFileSystemLoader {
```

Comment:
@jhsx Ha, it's not that easy:
```
can't load package: import cycle not allowed
package github.com/CloudyKit/jet
       	imports github.com/CloudyKit/jet/loaders/osfs
       	imports github.com/CloudyKit/jet
import cycle not allowed
package github.com/CloudyKit/jet
       	imports github.com/CloudyKit/jet/loaders/osfs
       	imports github.com/CloudyKit/jet
```

I'd have to put the Loader interface into a separate package, so it could be imported from both jet (template.go) and loaders.